pkg/cmd/roachtest/tests/cluster_to_cluster.go

```diff
@@ -67,11 +69,14 @@ type c2cSetup struct {
 	dst clusterInfo
 }
 
-func setupC2C(ctx context.Context, t test.Test, c cluster.Cluster) (*c2cSetup, func()) {
+func setupC2C(
+	ctx context.Context, t test.Test, c cluster.Cluster, srcKVNodes,
```

Comment:
nit: the last 2 params represent single nodes (or range boundaries) right? revise name to singular form - or perhaps something like `[src|dst]ClusterEndNode`?

Reply:
the params indicate number of nodes in src|dest cluster, which is indicated in their definitions. Leaving these as is.